codetransformer/transformers/precomputed_slices.py

```diff
@@ -0,0 +1,20 @@
+"""
```

Comment:
idk, it is about changing the constants of a function. I don't have a strong preference but I think they are related.